packages/fluentui/react-northstar/src/themes/teams/components/Card/cardStyles.ts

```diff
@@ -22,20 +22,30 @@ const cardStyles: ComponentSlotStylesPrepared<CardStylesProps, CardVariables> =
       width: v.width,
       height: v.height,
       boxShadow: v.boxShadow,
-      ...(p.size === 'small' && { width: v.sizeSmallWidth, height: v.sizeSmallHeight, padding: v.sizeSmallPadding }),
-      ...(p.size === 'large' && { width: v.sizeLargeWidth, height: v.sizeLargeHeight, padding: v.sizeLargePadding }),
-      ...(p.fluid && { width: v.fluidWidth, height: v.fluidHeight }),
-      ...(p.horizontal && { flexDirection: 'row' }),
-      ...(p.compact && { padding: v.compactPadding }),
-      ...(p.centered && { alignItems: 'center' }),
       ':hover': {
         boxShadow: v.boxShadowHover,
       },
-
+      ...(p.actionable && {
+        cursor: 'pointer',
+        ':focus': {
```

Comment:
```suggestion
        ':focus-visible': {
```

Should be this `focus-visible`? Can we add `.steps` file that will check focus styles?